packages/gcp-logging/src/index.ts

```diff
@@ -80,13 +80,30 @@ export const customLogger = (
         [k]: v,
       }))
 
+      let messageOrCause = message
+      let exception: any = undefined
+
+      if (cause) {
+        messageOrCause = message ?? Cause.pretty(cause)
+        switch(cause._tag){
```

Comment:
This won't cover for `interrupt`, `Parallel` and `Sequential`, do we want to set a default for debugging in case we get some of those?

ref: https://github.com/Effect-TS/effect/blob/4693101f6ac708d6b58b9ac49d4961aaf4752b13/src/internal/opCodes/cause.ts#L5

Reply:
Can't we have the default that is the same as for Fail, just Cause.pretty? What would be the downside?